sip-servlets-impl/src/main/java/org/mobicents/servlet/sip/address/ParameterableImpl.java

```diff
@@ -154,8 +158,29 @@ public void setParameter(String name, String value) {
 		//Fix from abondar for Issue 494 and angelo.marletta for Issue 502      
 		this.parameters.put(name.toLowerCase(), value);
 		if(header != null) {
+			boolean isQuotableParameter = false;
+			SipContext context = SipApplicationSessionCreationThreadLocal.lookupContext();
+			if (context != null){
+				Object object = context.getServletContext().getAttribute("org.restcomm.servlets.sip.QUOTABLE_PARAMETER");
+				if (object != null){
+					String quotableParameters = object.toString();
+					if (!quotableParameters.isEmpty()){
+						String[] parameters = quotableParameters.split(",");

```

Comment:
could we isolate this logic in a private method, something like isQutoableParam()?